packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -101,12 +102,8 @@ export function createAutocompleteDom<TItem extends BaseItem>({
     getInputProps: propGetters.getInputProps,
     getInputPropsCore: autocomplete.getInputProps,
     autocompleteScopeApi,
-    onDetachedEscape: isDetached
-      ? () => {
-          autocomplete.setIsOpen(false);
-          setIsModalOpen(false);
-        }
-      : undefined,
+    // In detached mode we don't want to close the panel when hittin `Tab`.
+    onDetachedTab: isDetached ? noop : undefined,
```

Comment:
```suggestion
    isDetached,
```

And then use this in `Input.ts`.

Reply:
Sure, I got a brain fart >.< There you go 22ff4785.